File: 세현/대지.js
const fs = require("fs");

const input = require("fs")
  .readFileSync(process.platform === "linux" ? "/dev/stdin" : "./input.txt")
  .toString()
  .trim()
  .split("\n")
//   .map((e) => Number(e));
  .map((line) => line.split(" ").map(Number));

// console.log(input);

function solution(input) {
  const N = input[0];
  let minx = 10000, miny = 10000;
  let maxx = -10000, maxy = -10000;
  for (let i = 1; i <= N; i++) {
    minx = Math.min(input[i][0], minx);
    miny = Math.min(input[i][1], miny);
    maxx = Math.max(input[i][0], maxx);
    maxy = Math.max(input[i][1], maxy);
  }

  return (maxx - minx) * (maxy - miny);
}

console.log(solution(input));

File: 진우/8weeks/9063. 대지/대지.py
N = int(input())

x = []
y = []

for _ in range(N):
    a, b = map(int, input().split())
    
    x.append(a)
    y.append(b)

row = 0
col = 0

if N >= 2:
    row = max(x) - min(x)
    col = max(y) - min(y)

print(row * col)